index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Meryl Streep</title>
+  <link rel="stylesheet" href="css.stylesheet" type="text/css" media="screen">
+  <nav>
```

Comment:
All content that should be seen by the user in the browser window, like this `nav`, belongs in the `body` not the `head`.  Please move.  🔨